helm-charts/codetrans/README.md

```diff
@@ -31,7 +31,7 @@ helm install codetrans codetrans --set global.HF_TOKEN=${HFTOKEN} --set global.m
 
 ### IMPORTANT NOTE
 
-1. To use model `mistralai/Mistral-7B-Instruct-v0.3`, you should first goto the [huggingface model card](https://huggingface.co/mistralai/Mistral-7B-Instruct-v0.3) to apply for the model access first. You need to make sure your huggingface token has at least read access to that model.
+1. To use model `Qwen/Qwen2.5-Coder-7B-Instruct`, you should first goto the [huggingface model card](https://huggingface.co/mistralai/Mistral-7B-Instruct-v0.3) to apply for the model access first. You need to make sure your huggingface token has at least read access to that model.
```

Comment:
Please fix also the model link.  And as Qwen model does not seem to require agreement, maybe that part should be change to non-rendered quote (so that it can resurrected when model changes back to one that requires agreement).

Reply:
Removed this line for this change. We can revert this PR if the model comes back.